Rakefile

```diff
@@ -265,6 +266,43 @@ See: #{latest_release_url}#{latest_release_anchor}
       x_client.post("tweets", tweet_body.to_json)
     end
   end
+
+  desc "Release to Arch Linux"
+  task :arch_linux do
+    releases_url = "https://api.github.com/repos/groonga/groonga/releases"
+    latest_released_version = URI(releases_url).open do |input|
+      JSON.parse(input.read)[0]["tag_name"].delete_prefix("v")
+    end
+    sha512_url = "https://github.com/groonga/groonga/releases/download/" +
+                 "v#{latest_released_version}/" +
+                 "groonga-#{latest_released_version}.tar.gz.sha512"
+    sha512 = URI(sha512_url).open do |input|
+      input.read.split[0]
+    end
+    pkgbuild = File.expand_path("ci/arch-linux/PKGBUILD")
+    pkgbuild_content = File.read(pkgbuild)
+    pkgbuild_content.gsub!(/^pkgver=.*$/) {"pkgver=#{latest_released_version}"}
+    pkgbuild_content.gsub!(/^pkgrel=.*$/) {"pkgrel=1"}
+    pkgbuild_content.gsub!(/^  "\h{128}"$/) {"  \"#{sha512}\""}
+    File.write(pkgbuild, pkgbuild_content)
+    sh("git", "add", pkgbuild)
+    sh("git", "commit", "-m", "arch-linux: Update to #{latest_released_version}")
+    sh("git", "push")
+
+    Dir.mktmpdir do |dir|
+      cd(dir) do
+        sh("git", "clone", "ssh://aur@aur.archlinux.org/groonga.git")
+        cd("groonga") do
+          cp(pkgbuild, "./")
+          sh("makepkg", "--printsrcinfo", out: ".SRCINFO")
+          sh("git", "add", "PKGBUILD", ".SRCINFO")
+          sh("git", "commit", "-m", "groonga-#{latest_released_version}-1")
+          sh("git", "log", "-p")
```

Comment:
Does `git log -p` show all logs?
Wouldn't `git log -p -1` be better?

Reply:
Ah, sorry. It's a debug print. I'll remove it.